codegen-sbt/src/sbt-test/compiletime-codegen/client-and-package/modules/client/src/main/scala/io/guizmaii/poc/caliban/client/Client.scala

```diff
@@ -0,0 +1,32 @@
+package io.guizmaii.poc.caliban.client
```

Comment:
Was this left over from when you moved the test repo over into the scripted roots?

Reply:
Yes, I tried to keep the test projects as close as my POC project so it's easy for me to backport things from my POC  project which I use to test my code.